app/components/todo.hbs

```diff
@@ -1,11 +1,29 @@
-<form onsubmit={{this.sumbit @model}}>  
-    <input type="text" value={{this.text}} onchange={{this.onChange}}/>
-    <button type="submit">Add To-Do</button>
-</form>
+<div>
+    <input
+      type="text"
+      value={{this.text}}
+      {{on "change" this.onChange}}
```

Comment:
In modern Ember, you attach event handlers to elements using the `{{on}}` modifier, not with `oneventname` attributes.

* [Handling events](https://guides.emberjs.com/release/components/component-state-and-actions/#toc_html-modifiers-and-actions)
* [API docs](https://api.emberjs.com/ember/4.7/classes/Ember.Templates.helpers/methods/on?anchor=on)